plugins/foreman_templates/4.0/index.md

```diff
@@ -0,0 +1,185 @@
+      
+# 3. Configuration
+
+After the installation there are new settings available. These are the default setting for both importing and exporting tasks. They can be overridden on each synchronization run and serve only as default so users don't have to specify the same value on every run. You will find them under Administer -> Settings -> TemplateSync. Only administrator can modify their values. Following table explains the attribute behavior. Note that some are only used on import or export actions.
+
+<table class="table table-bordered table-condensed table-striped">
+  <tr>
+    <th>Setting name</th>
+    <th>Meaning on importing</th>
+    <th>Meaning on exporting</th>
+  </tr>
+  <tr>
+    <td><i>associate</i></td>
+    <td>associate OS, Organization, Location based on metadata</td>
+    <td>N/A</td>
+  </tr>
+  <tr>
+    <td><i>branch</i></td>
+    <td colspan="2">git branch to read or write from/to (only for git url), default value is based on current Foreman version - e.g. develop or 1.15-stable</td>
+  </tr>
+  <tr>
+    <td><i>dirname</i></td>
+    <td>subdirectory under $repo to read from</td>
+    <td>subdirectory under $repo to export to</td>
+  </tr>
+  <tr>
+    <td><i>filter</i></td>
+    <td>import only templates with name matching this regular expression, after $prefix was applied</td>
+    <td>export only templates with name matching this regular expression</td>
+  </tr>
+  <tr>
+    <td><i>metadata_export_mode</i></td>
+    <td>N/A</td>
+    <td>one of refresh/remove/keep, refresh generates new metadata on export based on current associations and attributes, remove strips all metadata from template, keep keeps the same metadata that are part of template code</td>
+  </tr>
+  <tr>
+    <td><i>negate</i></td>
+    <td colspan="2">negates the $filter condition</td>
+  </tr>
+  <tr>
+    <td><i>prefix</i></td>
+    <td>adds specified string to beginning of the template on import, but only if the template name does not start with the prefix already</td>
+    <td>N/A</td>
+  </tr>
+  <tr>
+    <td><i>repo</i></td>
+    <td colspan="2">target path, specifies also the protocol, e.g. /tmp/dir or git://example.com, https://example.com, ssh://example.com</td>
+  </tr>
+  <tr>
+    <td><i>verbose</i></td>
+    <td colspan="2">adds extra verbose messages to the action output</td>
+  </tr>
+</table>
+
+Settings above are preconfigured for importing from upstream community-templates repository.
+
+# 4. Usage (features description)
+
+Both importing and exporting is available as rake tasks. Rake tasks are triggered from terminal. If you installed the plugin using foreman-installer, or simply it's package based installation), you should be able to run rake using `foreman-rake` command. If you installed Foreman alternatively, e.g. cloned from git, you should likely use `RAILS_ENV=production bundle exec rake` ran from your Foreman directory. There are other ways to run the action discussed under API chapter.
```

Comment:
I'd put the API sentence on a new line/paragraph to make it stand out for those who want to know.